content/docs/studio/get-started.md

```diff
@@ -0,0 +1,71 @@
+# Get started
```

Comment:
```suggestion
# Get Started with DVC Studio
```
So that search result snippets/ embeds are different than for https://dvc.org/doc/start